ci-operator/step-registry/addon/aws/osd/addon-aws-osd-workflow.yaml

```diff
@@ -0,0 +1,30 @@
+workflow:
+  as: addon-aws-osd
+  steps:
+    env:
+      API_HOST: stage
+      CHANNEL_GROUP: candidate
+      COMPUTE_MACHINE_TYPE: "m5.4xlarge"
+      MACHINE_POOL_INSTANCE_TYPE: "m5.2xlarge"
+      COMPUTE_NODES: "3"
+      PARALLEL: "true"
+      MULTI_AZ: "true"
+      DISABLE_WORKLOAD_MONITORING: "false"
```

Comment:
is this relevant for osd?

Reply:
no, it is false by default anyway.
thanks for pointing out